lib/cron/cache-pools-ipfs.ts

```diff
@@ -0,0 +1,118 @@
+import pinataSDK from '@pinata/sdk';
+import { ChainId, SubgraphProvider } from '@uniswap/smart-order-router';
+import { EventBridgeEvent, ScheduledHandler } from 'aws-lambda';
+import { Route53, STS } from 'aws-sdk';
+import { default as bunyan, default as Logger } from 'bunyan';
+import fs from 'fs';
+
+if (process.env.STAGE == 'beta') {
```

Comment:
You shouldn't really need this since you only create the lambda in Cloudformation if we are on beta but I get it if you added just for insurance. 

In any case, I think having the if statement surrounding everything like this would cause issues if we had unit tests as the module locally. 

I'd just put this inside the handler and do an early return `if (process.env.STAGE != Stage.BETA) { //log something; return; }`